tests/ops/test_simple_gla.py

```
@@ -329,6 +329,153 @@
+    tri_dh0, h0.grad = h0.grad.clone(), None
+
+    assert_close('  o', ref, tri, 0.004)
+    assert_close(' ht', ref_ht, tri_ht, 0.005)
+    assert_close(' dq', ref_dq, tri_dq, 0.005)
+    assert_close(' dk', ref_dk, tri_dk, 0.005)
+    assert_close(' dv', ref_dv, tri_dv, 0.005)
+    assert_close(' dg', ref_dg, tri_dg, 0.005)
+    assert_close('dh0', ref_dh0, tri_dh0, 0.005)
+
+
+@pytest.mark.parametrize('N', test_b_list)
```

Ruff (0.11.9)

Undefined name `test_b_list`

(F821)